README.md

```diff
@@ -1,2 +1,153 @@
 # OHSNAP-Documentation
```

Comment:
I would retitle this document to something like "Software/Network Configuration Documentation" to differentiate it from other docs we will produce (e.g. electrical engineering, manufacturing, tamper detection, etc.)

Reply:
Done!